packages/maskbook/src/database/Plugin/index.ts

```diff
@@ -59,3 +59,11 @@ export const createPluginDBAccess = db
 export function toStore(plugin_id: string, value: unknown): InStore {
     return { plugin_id, value }
 }
+
+export interface Plugin {
+    id: string
+    name: string
+    logo: string
```

Comment:
```suggestion
    logoURL: string
```

Reply:
e.g:
```jsx
import TokenIcon from "..."

{
// ...
    logo: <TokenIcon />
// ...
}
```

Reply:
> why is string type?

It could be a url to logo image of the plugin?

Reply:
```
logo: <img src={...} />
```

Reply:
I think it should represent raw data rather than UI related.